Publish OSOR interview about OpenZaak (#69)
open-zaak/open-zaak-website: src/pages/news/2021-04-26-OSOR-interview.en.md

---
path: "/en/news/2021-04-26-OSOR-interview"
date: "2021-04-26"
title: "OpenZaak featured in the European Open Source Observatory (OSOR)"
lang: en
template: default
---

# OpenZaak featured in the European Open Source Observatory (OSOR)

> The European portal [OSOR](https://joinup.ec.europa.eu/collection/open-source-observatory-osor), Open Source Observatory, published on April 7th an interview with Lazo Bozarov, Senior Supply Manager Digital Services at Utrecht Municipality about OpenZaak.

Reviewer: ```suggestion
The European portal [OSOR](https://joinup.ec.europa.eu/collection/open-source-observatory-osor), Open Source Observatory, published an interview with Lazo Bozarov, Senior Supply Manager Digital Services at Utrecht Municipality about OpenZaak on April 7th.
```

Author: Does it really matter?

Reviewer: Not semantically, but if we follow the journalistic method of leading with the most important message, I would say the date is the least important piece of information in this sentence.

> In this text, Bozarov comments on the advantages that working in open source provides once the new [Dutch API (Application Programmable Interface) standard for case-oriented](https://www.vngrealisatie.nl/nieuws/api-standaarden-zaakgericht-werken-vanaf-1-april) has been implemented.

Reviewer: Is a word missing here? Or does "case management" fit better than "case-oriented"?

Author: I've changed it!

> In this text, Bozarov comments on the advantages that working in open source provides once the new [Dutch API (Application Programmable Interface) standard for case-management](https://www.vngrealisatie.nl/nieuws/api-standaarden-zaakgericht-werken-vanaf-1-april) has been implemented.

Reviewer: ```suggestion
In this text, Bozarov comments on the advantages that working in open source provides once the new [Dutch API (Application Programmable Interface) standard for case management](https://www.vngrealisatie.nl/nieuws/api-standaarden-zaakgericht-werken-vanaf-1-april) has been implemented.
```

> You can read the interview [here](https://joinup.ec.europa.eu/collection/open-source-observatory-osor/news/we-liberated-our-own-data-silos-market).

Reviewer: ```suggestion
[Read the interview on Joinup](https://joinup.ec.europa.eu/collection/open-source-observatory-osor/news/we-liberated-our-own-data-silos-market).
```
